code/game/objects/structures/pit.dm

```diff
@@ -201,3 +201,17 @@
 	var/died = max(cur_year - rand(0,70),born)
 
 	message = "Here lies [nam], [born] - [died]."
+
+/obj/structure/gravemarker/attackby(obj/item/W, mob/user)
+	if(IS_HATCHET(W))
+		visible_message("<span class = 'warning'>\The [user] starts hacking away at \the [src] with \the [W].</span>")
+		if(do_after(user, 30))
```

Comment:
Is this an accidental inclusion? If not, it should probably be using do_tool_interaction since you're already in the area.

Reply:
yeah, its crap leftover from the rebase. Going to get rid of it.